apps/site/src/app/schedule/components/ShiftingCountdown/ShiftingCountdown.tsx

```diff
@@ -0,0 +1,158 @@
+"use client";
+
+import { AnimatePresence, motion } from "framer-motion";
+import { useEffect, useRef, useState } from "react";
+import BarLoader from "../BarLoader";
+import clsx from "clsx";
+import { Fireworks } from "@fireworks-js/react";
+
+import styles from "./ShiftingCountdown.module.scss";
+
+// NOTE: Change this date to whatever date you want to countdown to :)
+const HACKING_STARTS = "2024-01-26T18:00:00.000";
+const HACKING_ENDS = "2024-01-28T24:00:00.000";
```

Comment:
Issue [non-blocking]: 24:00:00 isn't a valid time, but we can update both of these later.